.github/ISSUE_TEMPLATE/improve_docs.yml

```diff
@@ -0,0 +1,33 @@
+name: "Documentation Issue"
+description: Issues related to documentation.
+title: "docs: "
+labels: [doc]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        _The more information you share, the faster we can help you._
+
+        Prior to opening the issue, please make sure that you:
+
+        - Use English to communicate.
+        - Search the [open issues](https://github.com/apache/apisix-ingress-controller/issues) to avoid duplicating the issue.
+
+  - type: textarea
+    id: current-state
+    attributes:
+      label: Current State
+      description: Describe the current state of the documentation.
+      placeholder: |
+        The documentation for the API in this page (url) is missing ...
```

Comment:
For the function?

Reply:
Do you mean to replace "documentation for the API" with "documentation for the function"? It is not clear to me.

Reply:
But in the Ingress controller project there is actually no API, only specific functions

Reply:
@navendu-pottekkat if you have time, please update this, thanks

Reply:
Updated.